lizard_auth_server/models.py

```diff
@@ -746,3 +740,136 @@ def as_dict(self):
             "organisation": self.organisation.as_dict(),
             "role": self.role.as_dict()
         }
+
+
+# New models
+# ##########
+
+class Profile(models.Model):
+    """Replacement for UserProfile."""
+    user = models.OneToOneField(
+        User,
+        verbose_name=_('user'),
+        related_name='profile')
+    company = models.ForeignKey(
+        'Company',
+        verbose_name=_('company'),
+        blank=True,
+        null=True)
+    created_at = models.DateTimeField(
+        verbose_name=_('created on'),
+        auto_now_add=True)
+    updated_at = models.DateTimeField(
+        verbose_name=_('updated on'),
+        auto_now=True)
+
+    def __str__(self):
+        return self.user.username
+
+    class Meta:
+        verbose_name = _("user profile")
+        verbose_name_plural = _("user profiles")
+
+    @property
+    def username(self):
+        return self.user.username
+
+    @property
+    def full_name(self):
+        return self.user.get_full_name()
+
+    @property
+    def first_name(self):
+        return self.user.first_name
+
+    @property
+    def last_name(self):
+        return self.user.last_name
+
+    @property
+    def email(self):
+        return self.user.email
+
+
+class Company(models.Model):
+    """Replacement for Organisation."""
+    name = models.CharField(
+        verbose_name=_('name'),
+        max_length=255,
+        null=False,
+        blank=False,
+        unique=True)
+    unique_id = models.CharField(
+        verbose_name=_('unique id'),
+        max_length=32,
+        unique=True,
+        default=create_new_uuid)
```

Comment:
Functies zijn over het algemeen serializeerbaar. Die gen_key functie was *niet* serializeerbaar want die zag er zo uit:
```
def gen_key(...):
    def func(...):
        ...
    return func
````
Functies in functies zijn niet serializeerbeer. Daarom die klasse.